tools/toitlsp/lsp/compiler/file_server.go

```diff
@@ -16,58 +16,41 @@
 package compiler
 
 import (
-	"bufio"
 	"fmt"
 	"net"
+	"strconv"
 	"sync"
 
-	cpath "github.com/toitware/toit.git/toitlsp/lsp/compiler/path"
 	"go.uber.org/zap"
 )
 
-type FileSystem interface {
-	Read(path string) (File, error)
-	ListDirectory(path string) ([]string, error)
-	PackageCachePaths() ([]string, error)
+type FileServer interface {
```

Comment:
New interface.
Currently there is only one class implementing it (`PortFileServer`) but I will add a new one soon.